tests/inputFields.spec.ts

```diff
@@ -0,0 +1,88 @@
+import { test, expect } from "@playwright/test";
+
+test.beforeEach(async ({ page }) => {
+  await page.goto("/");
+  await page.locator('[title="pettypes"]').click();
+  await page.waitForTimeout(3000); // wait for API response
```

Comment:
Using a hardcoded waits in the code is highly discouraged. In most of the cases you don't need it, because Playwright automatically wait for the element.
For example in this case, the Locator Assertion on the next line will wait until the header of the page will be displayed. `click()` command also waits until the locator is available.

Sometimes you may need to wait for API response, but need to do it differently, dynamically waiting for a particular API. The hardcoded wait - is the last thing to use, when nothing else works and should be avoided, because it increases test flakiness.

Check the lessons about Auto-Waiting mechanism. I explain these concepts.

Reply:
got it, thanks!